utils/exporters/blender/addons/io_three/exporter/api/object.py

```diff
@@ -374,7 +378,7 @@ def matrix(obj, options):
         parent_inverted = obj.parent.matrix_world.inverted(mathutils.Matrix())
         return parent_inverted * obj.matrix_world
     else:
-        return AXIS_CONVERSION * obj.matrix_world
```

Comment:
Ah, I see. Maybe we just remove that then, since it was only half working before.